frontend/src/components/pages/ManageStoryTranslationsPage.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+
+type ManageStoryTranslationsPageProps = {
+  storyIdParam: string | undefined;
+  storyTranslationIdParam: string | undefined;
+};
+
+const ManageStoryTranslationsPage = () => {
```

Comment:
nit: since this is for a singular story translation, `ManageStoryTranslationPage`